.circleci/config.yml

```diff
@@ -31,15 +31,10 @@ jobs:
             pip install yapf
             yapf -dr . --exclude ui/node_modules/
       - run:
-          name: Run the ui linter
+          name: Run the UI linter
           command: |
               cd ui
               npm run lint -- --list-different
```

Comment:
Total separate, but we may want to add to the README the command you gave me to auto-fix linting issues?

Reply:
Good idea, done. I also added `--list-different` to package.json and took it out of .circleci/config.yaml.